src/main/java/com/csc530/sat/condition/IntegerNEQCondition.java

```diff
@@ -0,0 +1,70 @@
+package com.csc530.sat.condition;
```

Comment:
I dislike this package structure.

Make it com.csc530.sat.condition.int.*

And do the same for booleans.

BTW, you will need GT and LT